jujugui/static/gui/src/app/components/env-size-display/env-size-display.js

```diff
@@ -29,6 +29,11 @@ YUI.add('env-size-display', function() {
       };
     },
 
+    componentWillReceiveProps: function() {
```

Comment:
I can do that however I haven't been documenting the native React functions as it's much better documented in the react docs.